config/manifests/bases/cert-manager-operator.clusterserviceversion.yaml

```diff
@@ -69,7 +69,7 @@ spec:
       name: certmanagers.operator.openshift.io
       version: v1alpha1
   description: |
-    The cert-manager Operator for Red Hat OpenShift provides seamless support for [cert-manager v1.15.5](https://github.com/cert-manager/cert-manager/tree/v1.15.2), which automates certificate management.
+    The cert-manager Operator for Red Hat OpenShift provides seamless support for [cert-manager v1.16.4](https://github.com/cert-manager/cert-manager/tree/v1.16.4), which automates certificate management.
```

Comment:
@swghosh What about this field? Do we still want to pin a specific version, or just pointing to https://github.com/cert-manager/cert-manager/releases?

Reply:
We want users to know from the Catalog page at the time of installation about which upstream version of cert-manager they're installing. There were such requests in the past which required us to make this doc effort of putting it in the CSV description: so unfortunately we cannot get rid of this yet!

Reply:
Makes sense